streams/src/test/java/org/apache/kafka/streams/state/internals/CachingKeyValueStoreTest.java

```diff
@@ -215,11 +249,6 @@ public void shouldThrowIfTryingToDeleteFromClosedCachingStore() throws Exception
         store.delete("key");
     }
 
-    @Test
-    public void shouldReturnNullIfKeyIsNull() throws Exception {
```

Comment:
This makes the cache store consistent with all other stores that throw an exception on put/get if key is null.